feature/about/src/main/java/io/github/droidkaigi/confsched2019/about/ui/item/AboutSection.kt

```diff
@@ -6,10 +6,18 @@ import io.github.droidkaigi.confsched2019.about.R
 
 class AboutSection : Section() {
```

Comment:
Please fix the conflict 🙇 
And you can inject ActivityActionCreator here 🙏 
(master code is changed)
https://github.com/DroidKaigi/conference-app-2019/blob/69bb64988bcc87378ecd5f65c10ee648a07daa4d/feature/about/src/main/java/io/github/droidkaigi/confsched2019/about/ui/item/AboutSection.kt#L10